nixos/modules/services/web-servers/caddy/default.nix

```
@@ -376,7 +376,9 @@ in
         ReadWriteDirectories = cfg.dataDir;
         StateDirectory = mkIf (cfg.dataDir == "/var/lib/caddy") [ "caddy" ];
         LogsDirectory = mkIf (cfg.logDir == "/var/log/caddy") [ "caddy" ];
-        Restart = "on-abnormal";
+        Restart = "on-failure";
+        RestartPreventExitStatus = 1;
+        RestartSecs = "5s";
```

```suggestion
        RestartSec = "5s";
```

Typo (my bad, didn't catch that in my original review)
See https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#RestartSec=

Do you want to fix it or should I? :)

---

Pushed #269613